iojs.org/aliases.json

```diff
@@ -115,5 +115,18 @@
       "gibfahn@gmail.com",
       "hello@bnb.im",
       "rtrott@gmail.com"
+  ] },
+    { "from": "cve-request", "to": [
+      "michael_dawson@ca.ibm.com",
+      "info@bnoordhuis.nl",
+      "viewtech@gmail.com"
+  ] },
+    { "from": "cve-mitre-contact", "to": [
+      "michael_dawson@ca.ibm.com",
+      "info@bnoordhuis.nl",
+      "viewtech@gmail.com"
+  ] },
+    { "from": "cna-discussion-list", "to": [
+      "michael_dawson@ca.ibm.com"
```

Comment:
Do we want a single point of failure on this? (Maybe we do. I don't actually know. Asking. Seems odd.)

Reply:
I'm happy to have others volunteer....  This this one is informational and will result in 10 messages that most often don't require action per week I thought I'd just leave it as me for now.